CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Mapbox welcomes participation and contributions from everyone.
 
 #### Bug fixes and improvements
 - Made `rerouteController` argument in `MapboxNavigation#setRerouteController` nullable. Null can be passed to disable automatic rerouting. [#5977](https://github.com/mapbox/mapbox-navigation-android/pull/5977)
+- :warning: Added checks to `DirectionsRoute` and `NavigationRoute` mappers that restricts to map `NavigationRoute` to `DirectionsRoute` and vice versa for some new Directions API features (for now it's EV routing), because original `DirectionsRoute` response requires for these kind of features but one is missing through mapping process. [#6004](https://github.com/mapbox/mapbox-navigation-android/pull/6004)
```

Comment:
> Added checks to `DirectionsRoute` and `NavigationRoute` mappers

Maybe specify the methods? It's not really clear what mappers you're talking about. Or sth like "`DirectionsRoute`-to-`NavigationRoute` mapper (and the opposite one)".

that restricts -> that restrict

requires for these kind -> requires this kind

missing through -> missing due to?

Reply:
thanks, fixed